### PR TITLE
fixed problems

### PR DESCRIPTION
**Pull request by Rarycops**

### CODE REVIEW

File: Assignment2/rarycops-Miguel_Jorge.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+c:class:01 
+	c:includes s:sensor:029;
+	c:includes co:computer:101.
+s:sensor:029
+	s:hasMeasurement m:measurement:8401.
+co:computer:101
+	co:computer:101:hasOwner u:user:10A.
+m:measurement:8401
+	m:measurement:8401:hasTemperature 29;
+	m:measurement:8401:atTime "2010-06-12T12:00:12".
```
Comment (ocorcho) [inline Assignment2/rarycops-Miguel_Jorge.ttl:16]: This should have an ^^xsd:dateTime

File: Assignment2/rarycops-Miguel_Jorge.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@base <http://www.oeg-upm.net/>.
+@prefix c: <http://www.oeg-upm.net/class/>.
+@prefix s: <http://www.oeg-upm.net/sensor/>.
+@prefix co: <http://www.oeg-upm.net/computer/>.
+@prefix m: <http://www.oeg-upm.net/measurement/>.
+@prefix u: <http://www.oeg-upm.net/user/>.
+c:class:01
```
Comment (ocorcho) [inline Assignment2/rarycops-Miguel_Jorge.ttl:7]: Not recommended to have two ":" Use only one before the prefix and substitute the another one with a hyphen or alike.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/rarycops-Miguel_Jorge.ttl (hunk 2 above)
Comment (Rarycops) [reply Assignment2/rarycops-Miguel_Jorge.ttl]: solved

File: Assignment2/rarycops-Miguel_Jorge.ttl (hunk 1 above)
Comment (Rarycops) [reply Assignment2/rarycops-Miguel_Jorge.ttl]: solved

**Review by Rarycops — COMMENTED** [review]
all the problems are supposedly solved

**Pull request merged**